src/execution/collectFields.ts

```diff
@@ -171,14 +171,20 @@ function collectFieldsImpl(
     variableValues,
     runtimeType,
     operation,
+    maskSuggestions,
     visitedFragmentNames,
   } = context;
 
   for (const selection of selectionSet.selections) {
     switch (selection.kind) {
       case Kind.FIELD: {
         if (
-          !shouldIncludeNode(selection, variableValues, fragmentVariableValues)
+          !shouldIncludeNode(
+            selection,
+            variableValues,
+            fragmentVariableValues,
+            maskSuggestions,
```

Comment:
He, I knew I was forgetting about arg name suggestions when I removed this but I honestly didn't mind I would just either always suggest for the built-in directives or not at all